metrics/web/lib/auth/presentation/widgets/sign_in_option_button.dart

```diff
@@ -21,11 +21,17 @@ class SignInOptionButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final loginTheme = MetricsTheme.of(context).loginTheme;
-    final loginOptionStyle = loginTheme.loginOptionButtonStyle;
+    final metricsTheme = MetricsTheme.of(context);
+    final isLoading =
+        Provider.of<AuthNotifier>(context, listen: false).isLoading;
```

Comment:
Maybe it would be better to wrap the button below in a `Selector` with `isLoading` value?